migrate_game.rb

```diff
@@ -263,6 +274,19 @@ def repair(game, original_actions, actions, broken_action)
   elsif broken_action['type'] == 'dividend' and broken_action['entity_type'] == 'minor'
     actions.delete(broken_action)
     return
+  
+  elsif (broken_action['type'] != 'pass') &&
```

Comment:
Is this reasonable? The extra pass gets caught up in a few different ways but I'm wondering if this is too permissive